pvremove: add page (#14360)
tldr-pages/tldr: pages/linux/pvremove.md

# pvremove

> Remove LVM labels from physical volume(s).
> More information: <https://man7.org/linux/man-pages/man8/pvremove.8.html>.

> - Remove LVM label from a physical volume:

Reviewer: ```suggestion
- Remove a LVM label from a physical volume:
```

`sudo pvremove {{physical_volume}}`

- Display detailed output during the operation:

`sudo pvremove --verbose {{physical_volume}}`

> - Remove LVM label without asking for confirmation:

Reviewer: ```suggestion
- Remove a LVM label without asking for confirmation:
```

`sudo pvremove --yes {{physical_volume}}`

> - Forcefully remove LVM label:

Reviewer: ```suggestion
- Forcefully remove a LVM label:
```

> `sudo pvremove --verbose {{/dev/sdaXY}}`
> 
> - Remove a LVM label without asking for confirmation:
> 
> `sudo pvremove --yes {{/dev/sdaXY}}`
> 
> - Forcefully remove a LVM label:
> 
> `sudo pvremove --force {{/dev/sdaXY}}`
> 
> - Display output in JSON format:
> 
> `sudo pvremove --reportformat json {{/dev/sdaXY}}`

Reviewer 2: ```suggestion
`sudo pvremove --verbose {{/dev/sdXY}}`

- Remove a LVM label without asking for confirmation:

`sudo pvremove --yes {{/dev/sdXY}}`

- Forcefully remove a LVM label:

`sudo pvremove --force {{/dev/sdXY}}`

- Display output in JSON format:

`sudo pvremove --reportformat json {{/dev/sdXY}}`
```

> `sudo pvremove --verbose {{physical_volume}}`
> 
> - Remove a LVM label without asking for confirmation:
> 
> `sudo pvremove --yes {{physical_volume}}`
> 
> - Forcefully remove a LVM label:
> 
> `sudo pvremove --force {{physical_volume}}`
> 
> - Display output in JSON format:
> 
> `sudo pvremove --reportformat json {{physical_volume}}`

Reviewer 2: ```suggestion
`sudo pvremove --verbose {{/dev/sdaXY}}`

- Remove a LVM label without asking for confirmation:

`sudo pvremove --yes {{/dev/sdaXY}}`

- Forcefully remove a LVM label:

`sudo pvremove --force {{/dev/sdaXY}}`

- Display output in JSON format:

`sudo pvremove --reportformat json {{/dev/sdaXY}}`
```
